src/ar100/Makefile

```diff
@@ -0,0 +1,45 @@
+CROSS_PREFIX=or1k-linux-musl-
+dirs-y += src/generic src/ar100 lib/ar100
+
+CFLAGS		+= -O3
+CFLAGS		+= -fno-builtin
+CFLAGS		+= -fno-pie
+CFLAGS		+= -ffreestanding
+CFLAGS		+= -msfimm -mshftimm -msoft-div -msoft-mul
+CFLAGS		+= -Ilib/ar100
+CFLAGS_klipper.elf := $(CFLAGS) -T src/ar100/ar100.ld
+CFLAGS_klipper.elf += -Wl,--gc-sections -static
+CFLAGS_klipper.elf += -Wl,--no-dynamic-linker
+
+SFLAGS		 = -nostdinc -MMD
+SFLAGS		+= -Ilib/ar100
+
+# Add source files
+src-y += ar100/main.c ar100/gpio.c ar100/serial.c
+src-y += ar100/util.c ar100/timer.c
+src-y += generic/crc16_ccitt.c generic/timer_irq.c
+
+# Remove files that are not needed to save space
```

Comment:
I'm removing some files instead of adding them.

Reply:
Ah, okay.  Lets keep what you have for now.  In the future we can look to add new Kconfig symbols - maybe something like `HAVE_LCD_BITBANGING` and `HAVE_SOFTWARE_ONLY_SPI`.

-Kevin